release-gap-package

```diff
@@ -649,7 +649,7 @@ fi
 for f in ./*/*.htm* ; do
   sed \
     -e 's;href="../../../doc/;href="https://www.gap-system.org/Manuals/doc/;g' \
-    -e 's;href="../../../pkg/GAPDoc[^\/]*/doc/;href="http://www.math.rwth-aachen.de/~Frank.Luebeck/GAPDoc/doc/;gi' \
+    -e 's;href="../../../pkg/GAPDoc[^\/]*/doc/;href="http://www.math.rwth-aachen.de/~Frank.Luebeck/GAPDoc/doc/;g' \
```

Comment:
This would solve my problem and seems safe:
```suggestion
    -e 's;href="../../../pkg/GAPDoc[^\/]*/doc/;href="http://www.math.rwth-aachen.de/~Frank.Luebeck/GAPDoc/doc/;g' \
    -e 's;href="../../../pkg/gapdoc[^\/]*/doc/;href="http://www.math.rwth-aachen.de/~Frank.Luebeck/GAPDoc/doc/;g' \
```
Also
```suggestion
    -e 's;href="../../../pkg/\(GAPDoc\|gapdoc\)[^\/]*/doc/;href="http://www.math.rwth-aachen.de/~Frank.Luebeck/GAPDoc/doc/;g' \
```
should work, but I don't know how portable it is.

Reply:
@wilfwilson why does suggested change has the same line repeated twice?

Reply:
These lines aren't the same, they differ in their capitalisation.